egs/gale_arabic/s5b/conf/online_pitch.conf

```diff
@@ -0,0 +1,46 @@
+## This config is given by conf/make_pitch_online.sh to the program compute-and-process-kaldi-pitch-feats,
+## and is copied by steps/online/nnet2/prepare_online_decoding.sh and similar scripts, to be given
```

Comment:
@danpovey, pushed.
I am not sure if other configurations like decode_dnn.config is used by nnet3/chain or no. Not called  by steps/nnet3/decode.sh or any of the local scripts.